src/fn_selectors.cpp

```diff
@@ -140,11 +140,13 @@ namespace Sass {
 
             // TODO: Add check for namespace stuff
```

Comment:
This fix isn't the same as this TODO is it?

Reply:
OK, maybe @mgreter can chime in about it later

Reply:
I believe it's referring to css namespaces i.e. https://oreillymedia.github.io/Using_SVG/extras/ch03-namespaces.html

These were properly accounted for once upon a time. I expect the comment is as relevant as it has ever been.